lisp/ess-r-package.el

```diff
@@ -289,9 +312,13 @@ With prefix ARG ask for extra args."
   "Interface for `devtools::check()'.
 With prefix ARG ask for extra args."
   (interactive "P")
-  (ess-r-package-eval-linewise
-   "devtools::check(%s)\n" "Checking %s" arg
-   '("" (read-string "Arguments: " "vignettes = FALSE"))))
+  (let ((actions '("" (read-string "Arguments: " "vignettes = FALSE"))))
```

Comment:
Too much copy pasting. I would add a wrapper which would dispatch to `ess-r-package-eval-linewise`.

Reply:
I had a hard time coming up with a design for such a wrapper -- do you have an idea about how to do it nicely?